src/queries/getCitiesInfo.js

```diff
@@ -0,0 +1,20 @@
+const db_connection = require("../database/db_connection");
+
+module.exports=(id,cb)=> {
+  db_connection.query(
+    'select city_name, city_pop, city_area from city where id = $1',[id], (error, result)=>{
+    if (error){
+        cb(error);
+    }else {
```

Comment:
When you call back error here, you will response with html text, but the client expects JSON, so it would be either sending the error as a json, or the client has  to handle the  error when the response is not a json